### ISSUE TITLE
forwardSlash issue with httplistener

### ISSUE THREAD

**Issue by Maesla** [cdmvision/authentication-unity#2]
Hi,

First, thank you for this package, incredible work and very nice implementation.

I have an issue with urls without forward slashes

Some times, the redirect_url is given without forward slash. However, HttpListener mandatorily needs forward slash 

` Prefixes must end in a forward slash ("/")`[https://learn.microsoft.com/en-us/dotnet/api/system.net.httplistener?view=net-7.0](url)

For example, a _http://localhost/login_ will fail and cause an exception  because of the httplistener forward slash necessity.

However, this redirect_url is given by the server settings, so there's nothing from client we can do to ask for a forward slashed version http://localhost/login/ in that case

TLDR, my proporsal is to add the forward slash to the http listener

```csharp
using System;
using System.Net;
using System.Threading;
using System.Threading.Tasks;
using UnityEngine;

namespace Cdm.Authentication.Browser
{
    /// <summary>
    /// OAuth 2.0 verification browser that runs a local server and waits for a call with
    /// the authorization verification code.
    /// </summary>
    public class StandaloneBrowser : IBrowser
    {
        private TaskCompletionSource<BrowserResult> _taskCompletionSource;

        /// <summary>
        /// Gets or sets the close page response. This HTML response is shown to the user after redirection is done.
        /// </summary>
        public string closePageResponse { get; set; } = 
            "<html><body><b>DONE!</b><br>(You can close this tab/window now)</body></html>";

        public async Task<BrowserResult> StartAsync(
            string loginUrl, string redirectUrl, CancellationToken cancellationToken = default)
        {
            _taskCompletionSource = new TaskCompletionSource<BrowserResult>();

            cancellationToken.Register(() =>
            {
                _taskCompletionSource?.TrySetCanceled();
            });

            using var httpListener = new HttpListener();
            
            try
            {
                char forwardSlash = '/';
                if (!redirectUrl.EndsWith(forwardSlash))
                {
                    redirectUrl += forwardSlash;
                }
                
                httpListener.Prefixes.Add(redirectUrl);
                httpListener.Start();
                httpListener.BeginGetContext(IncomingHttpRequest, httpListener);

                Application.OpenURL(loginUrl);
                
                return await _taskCompletionSource.Task;
            }
            finally
            {
                httpListener.Stop();
            }
        }

        private void IncomingHttpRequest(IAsyncResult result)
        {
            var httpListener = (HttpListener)result.AsyncState;
            var httpContext = httpListener.EndGetContext(result);
            var httpRequest = httpContext.Request;
            
            // Build a response to send an "ok" back to the browser for the user to see.
            var httpResponse = httpContext.Response;
            var buffer = System.Text.Encoding.UTF8.GetBytes(closePageResponse);

            // Send the output to the client browser.
            httpResponse.ContentLength64 = buffer.Length;
            var output = httpResponse.OutputStream;
            output.Write(buffer, 0, buffer.Length);
            output.Close();

            _taskCompletionSource.SetResult(
                new BrowserResult(BrowserStatus.Success, httpRequest.Url.ToString()));
        }
    }
}
```

**Comment by ibrahimpenekli** [comment]
Hi @Maesla ,

Thank you pointing this out. That's a good idea!
You can do pull request to develop branch, then I'll merge it.

**Comment by ibrahimpenekli** [comment]
PR merged
